### PR TITLE
Code reorganization

### PR DESCRIPTION
**Pull request by jyeray**

### CODE REVIEW

File: src/clients/alternatives.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-require "update-alternatives/UI/main_dialog"
+require "alternatives/UI/main_dialog"
```
Comment (jreidinger) [inline src/clients/alternatives.rb]: it should be y2alternatives not just alternatives

File: src/lib/alternatives/UI/alternative_dialog.rb [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require "yast"
+require "ui/dialog"
+
+Yast.import "UI"
+Yast.import "Label"
+
+module Y2Alternatives
```
Comment (jreidinger) [inline src/lib/alternatives/UI/alternative_dialog.rb]: ruby convention for location it camel cased name of class, so here `Y2Alternatives::Dialog::AlternativeDialog` should be in `y2_alternatives/dialog/alternative_dialog`

File: src/lib/alternatives/UI/alternative_dialog.rb [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  module Dialog
+    # Dialog for displaying possible Choices for one particular Alternative.
+    class AlternativeDialog < UI::Dialog
```
Comment (ancorgs) [inline src/lib/alternatives/UI/alternative_dialog.rb]: The agreement was to not include "Dialog" as part of the class name for classes in the `Dialog` namespace (is redundant). It should be just `Y2Alternatives::Dialog::Alternative`

File: src/clients/alternatives.rb (hunk 1 above)
Comment (jyeray) [inline src/clients/alternatives.rb]: according to the comment below, it should be `y2_alternatives` right?

File: src/lib/y2_alternatives/dialog/alternative.rb [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+require "yast"
+require "ui/dialog"
+
+Yast.import "UI"
+Yast.import "Label"
+
+module Y2Alternatives
+  module Dialog
+    # Dialog for displaying possible Choices for one particular Alternative.
+    class AlternativeDialog < UI::Dialog
```
Comment (ancorgs) [inline src/lib/y2_alternatives/dialog/alternative.rb]: As already commented, this name is redundant. See https://github.com/yast/yast-update-alternatives/pull/23#discussion_r71298083

File: src/lib/y2_alternatives/dialog/main.rb [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+require "y2_alternatives/dialog/alternative"
+require "y2_alternatives/alternative"
+
+Yast.import "UI"
+Yast.import "Popup"
+
+module Y2Alternatives
+  module Dialog
+    # Dialog where all alternatives groups in the system are listed.
+    class MainDialog < UI::Dialog
```
Comment (ancorgs) [inline src/lib/y2_alternatives/dialog/main.rb]: Same than before, redundant name. See https://github.com/yast/yast-update-alternatives/pull/23#discussion_r71298083

**Comment by ancorgs** [comment]
`test/dialogs/` should be `test/dialog/`

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**